config.go

```diff
@@ -771,7 +771,9 @@ func LoadConfig(interceptor signal.Interceptor) (*Config, error) {
 		// If it's a parsing related error, then we'll return
 		// immediately, otherwise we can proceed as possibly the config
 		// file doesn't exist which is OK.
-		if _, ok := err.(*flags.IniError); ok {
+		_, isIniError := err.(*flags.IniError)
```

Comment:
The linter wants this to be in the new format, using `errors.As()`. We have a nice helper function for that, so it should just be: `isIniError := lnutils.ErrorAs[*flags.IniError](err)` (same below).

Reply:
Should be good now. I didn't want to change the error casting style. That `lnutils.ErrorAs` is pretty nice